pup_fiction.py

```diff
@@ -256,7 +256,7 @@ def decrypt_selfs(in_dir, out_dir, blacklist=None):
     for root, dirs, files in os.walk(in_dir):
         for filename in files:
             name, ext = os.path.splitext(filename)
-            if ext in [".self", ".skprx", ".suprx"] and filename not in blacklist:
+            if (ext in [".self", ".skprx", ".suprx"] or filename=="eboot.bin") and filename not in blacklist:
```

Comment:
```suggestion
            if (ext in [".self", ".skprx", ".suprx"] or filename == "eboot.bin") and filename not in blacklist:
```
or maybe you'd want to make a `whitelist`